amqp_channel.c

```diff
@@ -455,44 +421,37 @@ PHP_METHOD(amqp_channel_class, setPrefetchSize)
 get the number of prefetches */
 PHP_METHOD(amqp_channel_class, getPrefetchSize)
 {
-	amqp_channel_object *channel;
+	amqp_channel_object *channel = AMQP_CHANNEL_OBJ_P(getThis());
 
-	if (zend_parse_parameters(ZEND_NUM_ARGS() TSRMLS_CC, "")) {
+	/* Parse out the method parameters */
+	if (zend_parse_parameters_none() == FAILURE) {
 		return;
 	}
 
-	/* Get the channel object out of the store */
-	channel = (amqp_channel_object *)zend_object_store_get_object(getThis() TSRMLS_CC);
-
 	RETURN_LONG(channel->prefetch_size);
 }
 /* }}} */
 
-
-
 /* {{{ proto amqp::qos(long size, long count)
 set the number of prefetches */
 PHP_METHOD(amqp_channel_class, qos)
 {
-	zval *id;
-	amqp_channel_object *channel;
+	amqp_channel_object *channel = AMQP_CHANNEL_OBJ_P(getThis());
 	amqp_connection_object *connection;
-	long prefetch_size;
-	long prefetch_count;
+	zend_long prefetch_size;
```

Comment:
Prefetch size and prefetch in rabbitmq-c defined as `uint32_t prefetch_size, uint16_t prefetch_count`, so we probably should take care of them here.